R/extract.R

```diff
@@ -0,0 +1,33 @@
+##' Extract touchstone-relevant data from sources. This should be called with
+##' \code{stoner::extract(path, con}, from within the extract code of a
```

Comment:
if `path, con` as a pair of bits of data that must be passed together, then you might want to have an "import object" that is just a list with these two bits of data together in a list with a class attribute. that's less chance for error

Reply:
sounds sensible